### PR TITLE
Issue 3294: ZCL_ABAPGIT_TIME: Unit Test fails on AS ABAP 752, SP04

### LINKED ISSUE #3294: ZCL_ABAPGIT_TIME: Unit tests fail on NW 752 SP04 dev
image

### PR DESCRIPTION
**Pull request by mariusraht2**
Closes #3294

### CODE REVIEW

File: .github/pull.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: "1"
```
Comment (larshp) [inline .github/pull.yml:1]: what is this file used for? The title and description of the PR indicates it fixes a unit test, but this file seems unrelated

Comment (mariusraht2) [reply .github/pull.yml:1]: Yes, totally not abapgit.. will remove it

**Comment by larshp** [comment]
hi,
it doesnt work on my test box, but it might also be messed up wrt timezones

image

**Comment by christianguenter2** [comment]
Works on my machine
image

**Comment by larshp** [comment]
hmm, perhaps just make the tests more simple, to avoid the flakyness, time is always difficult

**Comment by mariusraht2** [comment]
@larshp What do you suggest how we could make the tests more simple?

**Comment by larshp** [comment]
```abap
    zcl_abapgit_time=>get_utc( EXPORTING iv_unix = '1574605521'
                               IMPORTING ev_date = lv_date
                                         ev_time = lv_time ).

    cl_abap_unit_assert=>assert_not_initial( lv_date ).
    cl_abap_unit_assert=>assert_not_initial( lv_time ).
```

**Comment by larshp** [comment]
its not really perfect, but the code is used a lot, so errors will be found in it, better to spend time testing some of the more difficult stuff
